_content/gateways/thethingsindoor/faq.md

```diff
@@ -60,3 +60,15 @@ This is definitely not recommended since you cannot recreate a gateway with the
 
 If the TTIG is reset, only the WiFi and the LNS credentials will be deleted. It will retain its EUI and CUPS creds and will contact the CUPS server to obtain the LNS creds once the WiFi if configured.
 So reseting it (with the reset button) *will not affect* your registration of the gateway on the network.
+
+#### Q. I want to operate the gateway behind a firewall
+
+The following connections must be permitted in the firewall, the LNS address depends on your selected router in the Gateway configuration. 
+
+| IP Version | Protocol | Destination                                            | Port | Description |
+| ---------- | -------- | :----------------------------------------------------- | ---- | ----------- |
+| IPv4       | TCP      | lns.eu.thethings.network<br />lns.us.thethings.network | 443  | LNS         |
```

Comment:
Please use `lns.{eu|us|au|br|as}.thethings.network` to make this brief and future proof..  :) 
>  the LNS address depends on your selected router in the Gateway configuration. 

This is technically not true. All EU868 gateways currently only connect to EU and so on.

Reply:
OK, i have pushed the changes